RelVal/ReleaseValidation.C

```diff
@@ -216,18 +248,56 @@ void ProcessMonitorObjectCollection(
     } else if (clname.BeginsWith("TH")) {
       TObject *o = (TObject *)o2MonObjColl->FindObject(olname.Data());
       WriteHisto(o);
+      Counts++;
```

Comment:
Can you make this starting with a small "c"? It is a variable. I spotted this only with capital initial, if there are others, can you change them too?